### DISCUSSION TITLE
Actions on existing tiles

### DISCUSSION THREAD

**Discussion by MisterTdog** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#327] [Q&A] [answered]
Hi !
Nice work, I love it !
I would like to add actions on existing tiles.
As example, I want to reset timers when double_tap, or hold the Filters, Sensor, Bruches tiles.

I couldn't figured out how to do that...
Is it possible to achieve that?

I tried to set up a tile with the exact same name as the existing one. But it's appened. I would prefer to not manually rewrite all the existing tiles.

**Comment by PiotrMachowski** [comment, marked as answer]
Unfortunately it's not possible to modify autogenerated tiles. You can get their configuration by double-clicking "play" button: it will be printed in browser's console as JSON, you can convert it to YAML using https://www.json2yaml.com/

**Comment by MisterTdog** [comment]
Thank you for your prompt reply.
Unfortunately I dreamed about another answer ^^'
Any way. I may do that or find another way to reset filter. Thanks !
It could be an idea for further new feature ;)

**Reply by PiotrMachowski** [reply]
I don't think this will be added as a feature to be honest (because of the way how they are generated)
